hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/migration/NftStateTranslator.java

```diff
@@ -56,33 +56,33 @@ public static Nft nftFromMerkleUniqueToken(
                 .metadata(Bytes.wrap(merkleUniqueToken.getMetadata()));
 
         final var nftPrevIdPair = merkleUniqueToken.getPrev();
-        builder.ownerPreviousNftId(merkelUniqueTokenToUniqueTokenId(nftPrevIdPair));
+        builder.ownerPreviousNftId(merkelUniqueTokenToNftID(nftPrevIdPair));
 
         final var nftNextIdPair = merkleUniqueToken.getNext();
-        builder.ownerNextNftId(merkelUniqueTokenToUniqueTokenId(nftNextIdPair));
+        builder.ownerNextNftId(merkelUniqueTokenToNftID(nftNextIdPair));
```

Comment:
typo in method name `merkelUniqueTokenToNftID ` to `merkleUniqueTokenToNftID`

Reply:
Fixed